### PR TITLE
Initial adjustments

### PR DESCRIPTION
**Pull request by msavin**
Theodor - great chatting with you on the phone - my adrenaline is pumping. 

I'm sending you this pull request with some initial suggestions on how we could re-brand the package to highlight what makes it so beneficial. The first step is to re-organize the information, and then I will start another issue as to how we can expand it.

I would like to suggest moving the package itself into a folder called "package" with-in the folder. That way, we could create a neat separation between the documents and the actual code. I would do this but I would not want the contributors tab to show me adding 5k loc or something.

### CODE REVIEW

**Comment by msavin** [comment]
Preview of how the README would look: https://github.com/msavin/redis-oplog/blob/0d9e18aeaf51408a2c264f70cdd5247a0d61168a/README.md

**Comment by ramezrafla** [comment]
Thanks @msavin 

A quick grammatical note for the fun of it
"control it's reactive queries" --> "control its reactive" :)

**Comment by msavin** [comment]
@ramezrafla thanks - I just pushed a new update with spelling fixes and more general improvements. Let me know if you spot anything else.

File: docs/redis_unavailability.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## Redis Unavailability
+
+If redis server fails, it will `console.error` this fact, and it will keep retrying to connect every 30 seconds. Once connection is resumed
```
Comment (theodorDiaconu) [inline docs/redis_unavailability.md]: why are these separate readmes ?

Comment (theodorDiaconu) [reply docs/redis_unavailability.md]: I think it would be better if we do something like http://grapher.cultofcoders.com

Comment (theodorDiaconu) [reply docs/redis_unavailability.md]: Good point, but people will care about failover, so it can send also a positive signal

Comment (msavin) [reply docs/redis_unavailability.md]: What if we write "Designed to handle failover" as one of the features, and provide a link to more information?

BTW - can you get Pub/Sub to regress to polling the database whenever Redis becomes unavailable?

Comment (theodorDiaconu) [reply docs/redis_unavailability.md]: @msavin indeed sounds better.

To do that, it would require some big design changes.

**Pull request closed without merging**